lua/session_manager/config.lua

```diff
@@ -6,6 +6,7 @@ local config = {
     path_replacer = '__',
     colon_replacer = '++',
     autoload_last_session = true,
+    autoload_current_session = true,
```

Comment:
I would like to be more explicit:
```suggestion
    prefer_current_dir_session = true,
```

Reply:
i agree, do i need to do anything or is it automatically changes?(my first time doing a pull request)

Reply:
but maybe prefer is not the right way to describe it, since if there is no session for the current directory it will not load any session, while prefer implies it will load the last session

Reply:
Oh, I get. But maybe fallback to the latest session if the current directory doesn't have a session?

Reply:
I thought maybe to change the autoload from a boolean to a number to have:
0 - dont auto load
1 - only cwd
2 - prefer cwd, fallback to last
3 - last

it can also be done with the two booleans there are now, when both true is option number 2, but that might be harder to understand

Reply:
I agree. I would prefer to use Enum for it. Are you familiar with Plenary? It have [Plenary.Enum](https://github.com/nvim-lua/plenary.nvim/blob/master/lua/plenary/enum.lua).

Reply:
> do i need to do anything or is it automatically changes?(my first time doing a pull request)

For the future: yes, you need to press "Commit suggestion" to apply a suggestion.

Reply:
I thought a little more about this and I think the option 2 is not necessary.
I would suggest the following enumeration:

0 - dont auto load
1 - only cwd
3 - last

Because having two switches for it a little confusing.

Reply:
I did it, i'm not sure if i did it the right way, it is my first time with lua in general